Fails to load DLL on windows

**Describe the bug**
My pyatv GitHub actions jobs does not seem to work on Windows anymore. I get this error:

```
...
pyatv\support\chacha20.py:2: in <module>

    from chacha20poly1305_reuseable import ChaCha20Poly1305Reusable as ChaCha20Poly1305

E   ImportError: DLL load failed while importing chacha20poly1305_reuseable: %1 is not a valid Win32 application.
```
**To Reproduce**

In my case I just import the library on a Windows machine (`windows-latest` on GitHub in this case).
**Additional context**

https://github.com/postlund/pyatv/actions/runs/4728869785/jobs/8395410792?pr=1956

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -44,6 +44,7 @@ jobs:
         os:
           - ubuntu-latest
           - macOS-latest
+          - windows-latest
         extension:
           - "skip_cython"
           - "use_cython"

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -107,7 +107,7 @@ jobs:
         os:
           - macos-11
           - ubuntu-20.04
-          - windows-2022
+          - windows-2019
 
     steps:
       - uses: actions/checkout@v3
@@ -134,7 +134,7 @@ jobs:
           fetch-depth: 0
 
       - name: Install cibuildwheel
-        run: python3 -m pip install cibuildwheel==2.12.0
+        run: python3 -m pip install cibuildwheel==2.12.3
 
       - name: Build wheels
         run: python3 -m cibuildwheel --output-dir wheelhouse

